content/js/getting-started.md

```diff
@@ -0,0 +1,46 @@
+<a name="sign-up-portal"></a>
+## Signing up at Portal
+
+To start using the Skygear JS SDK, you need to register your account and
```

Comment:
Using Skygear JS SDK does not depend directly on the account on portal. It may be better saying:

"Before setting up the Skygear JS SDK, you need to have the Skygear server ready. You can sign up for one at the Skygear Developer Portal if you haven't got one."